draft-ietf-httpbis-http2bis.xml

```diff
@@ -3882,6 +3882,13 @@ cookie: e=f
           delimiters could be used to cause recipients to incorrectly interpret a message, which
           could be exploited by an attacker.
         </t>
+        <t>
+          <xref target="HttpHeaders"/> does not include specific rules for validation of
+          pseudo-header fields.  If the values of these fields are used, additional validation is
+          necessary.  This is particularly important where <tt>:scheme</tt>, <tt>:authority</tt>,
+          and <tt>:path</tt> are combined to form a single URI string (<xref target="RFC3986"/>).
```

Comment:
maybe add "and/or when combined with <tt>:method</tt>to rebuild a request line" since all implementations were caught with a space in that one.

Reply:
Thanks Willy, I've added a sentence.